app/helpers/authorization_helper.rb

```diff
@@ -0,0 +1,15 @@
+module AuthorizationHelper
+  include ActionView::Helpers::TranslationHelper
+
+  MODERATOR_SCOPES = %w[write_redactions].freeze
+
+  def authorization_scope(scope)
+    html = []
+    if MODERATOR_SCOPES.include? scope
+      html << image_tag("roles/moderator.png", :srcset => image_path("roles/moderator.svg", :class => "align-text-bottom"), :size => "20x20")
+      html << " "
```

Comment:
Putting the marker before the scope name is going to mean the scopes don't lineup properly which doesn't seem right - maybe it should go after the name?

Should we also add the administrator icon to privileged scopes?

Reply:
> Should we also add the administrator icon to privileged scopes?

Are they supposed to be used by administrators?

I added these stars in response to https://github.com/openstreetmap/openstreetmap-website/pull/4301#discussion_r1367689627 - to avoid accidentally granting moderator-only permissions to some app that might misuse them. Users are supposed to notice the star in the authorization dialog and be extra careful. `skip_authorization` privileged scope skips that dialog entirely. Users won't have a chance to review the permissions in this case.

The difference is that only administrators can register an app with privileged scopes but anyone can use the app with these scopes(?). Apps with moderator scopes can be registered by anyone, but only moderators can use features granted by these scopes.

Reply:
Basically do you want everyone who logged in at community.openstreetmap.org to see the admin star on https://www.openstreetmap.org/oauth2/authorized_applications next to "Read user email address"? Nobody gets to review that permission when they log in.

Reply:
Moved the star marker after the name.

Reply:
Thanks for moving the marker. You argument about the administrator case being differnet seems sensible so let's leave that.